### PR TITLE
Bug 1490992, added Privileges and Volume Builds guidance

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1490992

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@tim PTAL. Thanks!

**Comment by ahardin-rh** [comment]
@abhgupta Please review as well.

**Comment by ahardin-rh** [comment]
@openshift/team-documentation please peer review

**Comment by bfallonf** [comment]
@ahardin-rh LGTM!

File: creating_images/guidelines.adoc [hunk 1]
```diff
@@ -234,11 +234,19 @@ See the following references for other guidelines:
...
-
+[[openshift-specific-guidelines]]
 == {product-title}-Specific Guidelines
 The following are guidelines that apply when creating container images specifically
 for use on {product-title}.
-
+ifdef::openshift-online[]
+*Privileges and Volume Builds*
+
+Images cannot use the `VOLUME` directive. Images using a read/write file system
```
Comment (tiwillia) [inline creating_images/guidelines.adoc]: I think we might want to be a bit more explicit here: `Docker images cannot be built using the 'VOLUME' directive.`

Comment (wzheng1) [reply creating_images/guidelines.adoc]: Vote for this comment.

**Comment by ahardin-rh** [comment]
Verified by QE. No revision history needed, as this will appear in Online only (where no rev histories are built).

**Pull request merged**